tests/CloudinaryAdpterTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
The name of this file does not match the name of the class. There's a spelling error. Can you fix that?

Reply:
Ok on it